snippets/snippets.code-snippets

```diff
@@ -418,5 +418,61 @@
             "\t</abstract>",
             "</merge>"
         ]
+    },
+    "Substeps": {
+        "prefix": "ss",
+        "body": [
+            "<substeps>",
+            "\t<step>",
+            "\t\t<para>",
+            "\t\t\t${1:insert text}",
+            "\t\t</para>",
+            "\t</step>",
+            "\t<step>",
+            "\t\t<para>",
+            "\t\t\t${2:insert text}",
+            "\t\t</para>",
+            "\t</step>",
+            "</substeps>"
+        ],
+    },
+    "Keycap": {
+        "prefix": "kc",
+        "body": "<keycap>${1:insert keycap}</keycap>"
+    },
+    "Keycap with a function": {
+        "prefix": "kcf",
+        "body": "<keycap function=\"${1|enter,alt,shift,ctrl|}\"/>"
```

Comment:
Maybe add all possible values for `function`? According to the [`keycap` reference](https://tdg.docbook.org/tdg/5.2/keycap):

* `function` (enumeration)
    * “alt”
    * “altgr”
    * “backspace”
    * “command”
    * “control”
    * “delete”
    * “down”
    * “end”
    * “enter”
    * “escape”
    * “home”
    * “insert”
    * “left”
    * “meta”
    * “option”
    * “pagedown”
    * “pageup”
    * “right”
    * “return”
    * “shift”
    * “space”
    * “tab”
    * “up”